src/ci/docker/run.sh

```diff
@@ -36,12 +36,13 @@ elif [ -f "$docker_dir/disabled/$image/Dockerfile" ]; then
         echo Cannot run disabled images on travis!
         exit 1
     fi
-    retry docker \
+    # retry messes with the pipe from tar to docker. Not needed on non-travis
+    tar --transform 's/^\.\/disabled\//.\//' -C $docker_dir -c . | docker \
```

Comment:
The transform looks right, but it generates a very strange path for the dockerfile for me - `.\\/Dockerfile`. All of the backslashes are a bit crazy anyway, can you make `#` the escape character? i.e. `s#^\./a/#./#'`

Additionally, can you add a brief description line of what we're doing here? E.g. something like "The tar transform below makes the images in the disabled directory look like they're in the main images directory to `docker build`."